src/js/eventbuilder.ts

```diff
@@ -0,0 +1,61 @@
+import { resolvedSyncPromise } from '@sentry/utils';
+import { StackParser, Severity, SeverityLevel, EventHint, Event, Thread } from '@sentry/types';
+import { parseStackFrames } from '@sentry/browser/cjs/eventbuilder';
+
+export { eventFromException } from '@sentry/browser/cjs/eventbuilder';
```

Comment:
Should we export these in `@sentry/browser` or how can we import these private files?

Reply:
We should export these in `@sentry/browser` if we need them.

Reply:
https://github.com/getsentry/sentry-javascript/pull/6515